configs/machines/aleph.yaml

```diff
@@ -0,0 +1,124 @@
+# THE aleph.iccp.kr YAML config file
+
+name: aleph
+
+accounting: true
+batch_system: "pbs"
+
+operating_system: "SLES"
+
+jobtype: compute
+sh_interpreter: "/bin/sh"
+
+
+choose_jobtype:
+        post:
+                partition: serial
+        compute:
+                partition: iccp
+        tidy_and_resubmit:
+                partition: iccp
+
+choose_partition:
+        "iccp":
+                cores_per_node: 40
+        "serial":
+                cores_per_node: 1
+
+logical_cpus_per_core: 2
+
+threads_per_core: 1
+pool_dir: "/proj/awi"
+
+pool_directories:
+        pool: "/proj/awi"
```

Comment:
Not only for this file but I think that we need some form of consistency for the directory paths. I opt for using a directory separator (`/`) at the end of the paths. I realized that sometimes there are errors due to a missing slash at the end. What do you think @mandresm?

Reply:
Always saver to add them. I'll do so here.